zerver/lib/actions.py

```diff
@@ -638,8 +638,8 @@ def do_start_email_change_process(user_profile, new_email):
     activation_url = EmailChangeConfirmation.objects.get_link_for_object(obj, host=user_profile.realm.host)
     context = {'realm': user_profile.realm, 'old_email': old_email, 'new_email': new_email,
                'activate_url': activation_url}
-    send_email('zerver/emails/confirm_new_email', new_email, from_email=settings.DEFAULT_FROM_EMAIL,
-               context=context)
+    from_email = "Zulip Account Security <%s>" % (settings.DEFAULT_FROM_EMAIL,)
+    send_email('zerver/emails/confirm_new_email', new_email, from_email, context=context)
```

Comment:
I would still use the 
`from_email="Zulip Account ..." `
format here. 
Two reasons: we generally don't like declaring variables that only get used once, and could go in-line (adds unneeded indirection). Also, we require that keyword arguments use the keyword.